finetune_adapter.py

```diff
@@ -90,15 +86,16 @@ def train(
     optimizer: torch.optim.Optimizer,
     train_data: np.ndarray,
     val_data: np.ndarray,
-    checkpoint_dir: Path,
+    pretrained_dir: Path,
+    out_dir: Path,
```

Comment:
Can you update the https://github.com/Lightning-AI/lit-stablelm/blob/main/howto/finetune_adapter.md file to show out_dir usage?

If we don't have a single checkpoint, then we should recommend out_dir to have some sort of information of which checkpoint was used, like `out/adapter/stabilityai.../alpaca`

Alternatively, out_dir could be created inside `checkpoint_dir` and then it would be clear from the structure

Reply:
The single folder motivation stems from this review comment: https://github.com/Lightning-AI/lit-stablelm/pull/19#discussion_r1186457956 where it was suggested that we could use a single directory and infer the structure from it.

I understand the motivation for keeping a separate `out_dir` argument, but it could be created inside the "main" directory to be in line with the suggestion above.

Reply:
I agree, let's avoid having the finetuned output in the original checkpoint directory.